tests/ltp/patches/fix_recvmsg_recvmsg01.patch

```diff
@@ -69,27 +73,39 @@ index 13bcaa4e0..b5ea895f4 100644
  /* 7 */
  	{
  	PF_INET, SOCK_STREAM, 0, iov, -1, (void *)buf, sizeof(buf),
-@@ -246,6 +249,7 @@ int main(int argc, char *argv[])
+@@ -245,7 +248,7 @@ int main(int argc, char *argv[])
+ 	tst_exit();
  }
  
- pid_t pid;
-+pthread_t tid;
+-pid_t pid;
++pthread_t tid = NULL;
  char tmpsunpath[1024];
  
  void setup(void)
-@@ -268,8 +272,9 @@ void setup(void)
+@@ -263,13 +266,19 @@ void setup(void)
+ 
+ 	signal(SIGPIPE, SIG_IGN);
+ 
+-	pid = start_server(&sin1, &sun1);
++	start_server(&sin1, &sun1);
+ }
  
  void cleanup(void)
  {
 -	if (pid > 0)
 -		(void)kill(pid, SIGKILL);	/* kill server */
-+	close(ufd);
```

Comment:
@shaikshavali1 These three lines are very confusing. There is red and green at the same line. Is this a deleted line or added line?

Reply:
It looks like these lines are deleted.